ansible/roles/tileserver_step2/tasks/main.yml

```diff
@@ -63,6 +63,7 @@
       - 'maxspeed.xml'
       - 'signals.xml'
       - 'electrification.xml'
+      - 'gauge.xml'
```

Comment:
Can we put these 3 loops with the same keys in a variable and reuse it at the 3 places?

Reply:
done.